glide.lock

```diff
@@ -23,9 +23,14 @@ imports:
 - name: github.com/m3db/m3cluster
   version: b596a86b06f37c9150ea352d6ec1549ed19f304b
   subpackages:
+  - client
```

Comment:
Yea, for some reason I was getting vendor errors when trying to build master. Essentially some interfaces in m3cluster weren't being satisfied because they were using types from m3metrics vendor repo but the interfaces were expecting types from m3clusters vendor repo. When I did a glide update the errors were resolved and these changes were applied so I figured they were necessary.